packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -568,6 +568,8 @@ export const messages = {
   conflict_detect_no_differences: 'No differences',
   conflict_detect_diff_title: '%s//%s ↔ local//%s',
   conflict_detect_diff_command_title: 'Compare Files',
+  conflict_detect_remote_last_modified_date: 'Org last modified date: %s \n',
```

Comment:
@sbudhirajadoc Let us know what you think about this wording!

Reply:
We are thinking about renaming `last modified date` to `last deploy/retrieve date`. For example, it could be confusing when a user modifies something locally but finds their `Local last modified date` unchanged.

Reply:
Let's go with "local last sync date" and "Org last modified date". Let me know if you have any questions.

Reply:
Thanks, Sonal! Updated. ;D